EmojiArt/Document/Views/EmojiArtDocumentView.swift

```diff
@@ -49,6 +60,19 @@ struct EmojiArtDocumentView: View {
             .font(.system(size: Constants.defaultEmojiFontSize))
     }
 
+    private func didSelect(_ emoji: EmojiArtModel.Emoji) {
+        print("\(emoji.text) emoji has been selected")
```

Comment:
I would recommend switch to use debugPrint("message is here"), please check in google what is the difference